src/sagas.js

```diff
@@ -245,6 +277,80 @@ function* startVm (action) {
   yield stopProgress({ vmId: action.payload.vmId, name: 'start', result })
 }
 
+function* showEditVm (action) {
+  yield put(updateDialogType('edit'))
```

Comment:
Maybe I'm missing something, but I don't prefer saga to be used for opening a dialog.
It seems to be too procedural.

Much better approach for Edit VM Dialog can be via React itself, something like following pseudo-code:
  <EditVM vm={myVmToEdit} clusters={clusters} templates={allTemplates} ...  />

The EditVm component state consists of a VM described as references to other resources, similar to:
this.state = {
vm: {
name,
cpus,
clusterId,
templateId,
...
}
}

With this approach, we
- avoid of this procedural code
- leverage React/Redux for updates if underlying sub-resources are changed or gradually loaded
- move rendering-related code to React component

@jelkosz , @matobet , please comment.